### PR TITLE
CMGR-36289 rde public api docs

### PR DESCRIPTION
**Pull request by crysty13**
new APIs related to RDEs
small other changes.

## How Has This Been Tested?

tested locally according to the readme that shows how to start all locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: swagger-specs/api.yaml [hunk 1]
```diff
@@ -1375,6 +1377,44 @@ paths:
           description: Successful setting of variables
         '404':
           description: Environment not found
+  '/api/program/{programId}/environment/{environmentId}/reset':
+    put:
+      tags:
+        - Rapid Development Environments
+      summary: Reset Rapid Development Environment
```
Comment (bohnertchris) [inline swagger-specs/api.yaml:1384]: Resets not reset

File: swagger-specs/api.yaml [hunk 2]
```diff
@@ -1375,6 +1377,44 @@ paths:
...
+          required: true
+          type: string
+        - name: environmentId
+          in: path
+          description: Identifier of the environment
+          required: true
+          type: string
+        - name: x-gw-ims-org-id
+          in: header
+          description: IMS organization ID that the request is being made under.
```
Comment (bohnertchris) [inline swagger-specs/api.yaml:1400]: Remove period

Comment (crysty13) [reply swagger-specs/api.yaml:1400]: all descriptions seem to have period. please confirm you want the period to be removed.

Comment (bohnertchris) [reply swagger-specs/api.yaml:1400]: Thanks for asking. Leave it for now. I will create a PR to remove all of them en masse.

**Pull request merged**